### ISSUE TITLE
Installation failed, reverting ./composer.json and ./composer.lock to their original content.

### ISSUE THREAD

**Issue by Elusoji** [artesaos/seotools#269]
please getting this issue i am using laravel 9
```
Problem 1
    - Root composer.json requires artesaos/seotools ^0.21.0 -> satisfiable by artesaos/seotools[v0.21.0].
    - artesaos/seotools v0.21.0 requires illuminate/config 5.8.* || ^6.0 || ^7.0 || ^8.0 -> found illuminate/config[v5.8.0, ..., 5.8.x-dev, v6.0.0, ..., 6.x-dev, v7.0.0, ..., 7.x-dev, v8.0.0, ..., 8.x-dev] but these were not loaded, likely because it conflicts with another require.

**Comment by Bjornftw** [comment]
Seotools is not ready for Laravel 9 yet. See #267

**Comment by vinicius73** [comment]
Closed by #267
